resources/js/features/games/components/GameSidebarFullWidthButtons/GameSidebarFullWidthButtons.tsx

```diff
@@ -0,0 +1,106 @@
+import type { FC } from 'react';
+import { useTranslation } from 'react-i18next';
+import { LuBookText, LuFileCode, LuFileText, LuSearch, LuTickets, LuWrench } from 'react-icons/lu';
+
+import { GameCreateForumTopicButton } from '@/common/components/GameCreateForumTopicButton';
+import { PlayableOfficialForumTopicButton } from '@/common/components/PlayableOfficialForumTopicButton';
+import { PlayableSidebarButton } from '@/common/components/PlayableSidebarButton';
+import { usePageProps } from '@/common/hooks/usePageProps';
+
+interface GameSidebarFullWidthButtonsProps {
+  game: App.Platform.Data.Game;
+}
+
+export const GameSidebarFullWidthButtons: FC<GameSidebarFullWidthButtonsProps> = ({ game }) => {
+  const { auth, can, numCompatibleHashes, numOpenTickets } =
+    usePageProps<App.Platform.Data.GameShowPageProps>();
+
+  const { t } = useTranslation();
+
+  const canShowEssentialResources = numCompatibleHashes > 0 || game.forumTopicId;
+  const canShowExtras = !!auth?.user;
+  const canShowManagement = can.manageGames;
+
+  if (!canShowEssentialResources && !canShowExtras && !canShowManagement) {
+    return null;
+  }
+
+  return (
+    <div className="flex flex-col gap-4">
+      {canShowEssentialResources ? (
+        <div className="flex flex-col gap-1">
+          <p className="text-xs text-neutral-300 light:text-neutral-800">
+            {t('Essential Resources')}
+          </p>
+
+          {numCompatibleHashes > 0 ? (
+            <PlayableSidebarButton
+              className="border-l-4 border-l-link"
+              href={route('game.hashes.index', { game: game.id })}
+              isInertiaLink={true}
+              IconComponent={LuFileText}
+              count={numCompatibleHashes}
+            >
+              {t('Supported Game Files')}
+            </PlayableSidebarButton>
+          ) : null}
+
+          <PlayableOfficialForumTopicButton game={game} />
+        </div>
+      ) : null}
+
+      {canShowExtras ? (
+        <div className="flex flex-col gap-1">
+          <p className="text-xs text-neutral-300 light:text-neutral-800">{t('Extras')}</p>
+
+          <PlayableSidebarButton
+            href={route('game.suggestions.similar', { game: game.id })}
+            isInertiaLink={true}
+            IconComponent={LuSearch}
+          >
+            {t('Find Similar Games')}
+          </PlayableSidebarButton>
```

Comment:
The old label wasn't really any better, but this is misleading. Most of the results returned aren't similar at all. In fact, we have a separate sidebar section specifically devoted to Similar Games.

Unfortunately, I don't have a better name for this. The page itself is labelled "Game Suggestions". Would "Find Related Games" work?

Reply:
I agree that "Find Related Games" is a better choice for a label. Updated in latest.